db/schema_tables.cc

```diff
@@ -60,7 +60,6 @@
 #include "mutation_query.hh"
 #include "system_keyspace.hh"
 #include "cql3/cql3_type.hh"
-#include "cdc/log.hh"
```

Comment:
It's not ok to break codebase with one commit and fix with another. Changes have to be bisectable - every commit has to build correctly and pass all the tests. Please rearrange the patches in such a way to keep every commit build and pass tests.

Reply:
Probably it's enough to apply this patch after switching to using extension.